content/game_process.tex

```diff
@@ -170,10 +170,18 @@ \subsection{Kick-off}
 The head referee signals the kick-off by a single whistle blow, followed by the call ``Playing''. The head referee must signal this from the T-junction of the halfway line.
 
 After the head referee has signaled the kick-off, the robot's state is switched to \emph{playing} by the GameController.
-The defensive team must stay outside the center circle until the ball is in play. The ball is in play once it is touched by the attacking team or once \qty{\KickOffBallFreeTime}{\second} have elapsed in the playing state. Before the ball is in play, robots are not allowed to leave their own halves. Otherwise, a robot will be penalized with the ``Illegal Position'' penalty~(\cf \cref{sec:illegal_positioning}).
+The defensive team must stay outside the center circle until the ball is free. Before the ball is free, robots are not allowed to leave their own halves and additionally defensive players are not allowed to enter the center circle. Otherwise, a robot will be penalized with the ``Illegal Position'' penalty~(\cf \cref{sec:illegal_positioning}).
+
+A goal may be scored directly from a kick-off against the opposing team. If the ball goes directly into the goal of the player taking the kick-off, a corner kick is awarded to the opposing team.
+
+\subsubsection{Ball free}
+The ball is free and kick-off ends once 
+\begin{itemize}
+  \item it is touched by the attacking team and moved clearly or
```

Comment:
The "moved clearly" part is quite a difference to previous rules where people were often obsessed with the tiniest contacts between the foot and the ball (even more so in penalty kicks) that aren't even noticeable for the robots. I would actually be happy if this wasn't the case anymore (i.e. I support including the "moved clearly" part") because it is more consistent with real soccer (but then for free kicks and penalty kicks as well).

Reply:
I agree except for penalty kick.

Reply:
Is now in the PR